src/shims/env.rs

```diff
@@ -400,16 +400,17 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
     }
 }
 
-// Local helper function to be used in Windows shims
-#[allow(non_snake_case)]
-fn HowWasBufferSize((success, len): (bool, u64)) -> u32 {
+/// Check whether an operation that writes to a target buffer was successful.
+/// Accordingly select return value.
+/// Local helper function to be used in Windows shims.
```

Comment:
This is another helper function that IMO should be at the top of the file, not at the bottom.